triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -1606,13 +1606,13 @@ object Runner {
       case Trigger.Level.High =>
         val result = for {
           inFlightCommands <- getInFlightCommands(svalue)
-          commands <- mapLookup(SText(commandId), inFlightCommands)
+          commands <- mapLookup(commandId, inFlightCommands)
           entries <- commands.expect("SList", { case SList(entries) => entries })
         } yield entries.toImmArray.toSeq.count { cmd =>
           isCreateCommand(cmd) || isCreateAndExerciseCommand(cmd)
         }
 
-        Some(result.orConverterException)
+        Some(result.getOrElse(0))
```

Comment:
Lookups can fail (e.g. because the command ID is not known) and so we should return 0 in such cases